.gitignore

```diff
@@ -40,6 +40,12 @@ yarn-error.log*
 # local env files
 .env
 
+
+*.idea
+*.iml
+
 # cypress
 cypress/screenshots
 cypress/videos
+cypress.env.json
```

Comment:
What is this file used for?

Reply:
Good question. I didn't know either, so I looked it up.

It's documented here: <https://docs.cypress.io/guides/guides/environment-variables.html#Option-2-cypress-env-json>

This was in Tanay's branch, and I just left it---I assume it came from some list of files that are good practice to put in a `.gitignore`.     I don't think we are currently using `cypress.env.json`, but if we ever were, it apparently should be in the `.gitignore`.